exts/fun.py

```diff
@@ -69,6 +69,24 @@ async def _funetics_lookup(self, ctx: commands.Context, *, msg: str):
         embed.colour = cmn.colours.good
         await ctx.send(embed=embed)
 
+    @commands.command(name="uwuify", aliases=["uwu"], category=cmn.Cats.FUN)
+    async def _uwuify(self, ctx: commands.Context, *, msg: str):
+        uwuified_text = ''
+        for i, c in enumerate(msg):
+            previous_char = msg[i - 1] if i > 0 else ''
+            if c == 'L' or c == 'R':
+                uwuified_text += 'W'
+            elif c == 'l' or c == 'r':
+                uwuified_text += 'w'
+            elif c == 'O' or c == 'o':
+                if previous_char == 'N' or previous_char == 'n' or previous_char == 'M' or previous_char == 'm':
+                    uwuified_text += "yo"
```

Comment:
for this, [`str.replace()`](https://docs.python.org/3/library/stdtypes.html#str.replace) or [`re.sub()`](https://docs.python.org/3/library/re.html#re.sub) would probably be cleaner.

Also, don't forget about `na` to `nya`.